integrationtests/pom.xml

```diff
@@ -32,6 +32,18 @@
             <version>1.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I think you'll be able to prune some dependency too, as you'll get them transitively from the junit5 extension.  Zookeeper and testcontainers should be removable.  You can probably remove the maven property that supplies the version number too.

Reply:
done. Thanks!